documentation/src/docs/getting-started.md

```diff
@@ -56,18 +56,6 @@ We can check and change the Flink version that Pravega builds against via the `f
 
 **Note**: Only Flink versions that are compatible with the latest connector code can be chosen.
 
-#### Building against another Scala version

```

Comment:
We just don't officially publish the scala 2.11 connector, but we can still leave this part here to teach users how to build a scala 2.11 connector locally.